scripts/configure-hypervisor.sh

```diff
@@ -318,7 +318,7 @@ generate_qemu_options() {
 
 	# Disable XEN driver
 	case "$arch" in
-	aarch64) ;;
+	aarch64) qemu_options+=(size:--disable-xen) ;;
```

Comment:
since this is the section where XEN is disabled, how about disable it for all architectures?

Reply:
sure